### DISCUSSION TITLE
Using email under a custom domain

### DISCUSSION THREAD

**Discussion by aralroca** [vercel/vercel#4155] [General] [answered]
I recently transferred my domain to Vercel. I want to ask if also is there a way to continue use domain emails as contact@mydomain.com? I took a look at the documentation but I didn't found anything.

Thank you so much 🙂

**Comment by coetry** [comment, marked as answer]
Hi! You can definitely do that by adding MX records via our [CLI](https://vercel.com/docs/now-cli#commands/dns)

We're well aware that adding DNS records via CLI us super unintuitive and so we're committed to building first-class support for this in our UI pretty soon. The CLI should get you going for now though!

**Reply by aralroca** [reply]
Yes! Thank you and sorry, because I already did it in this way finally without notify here! The CLI is cool! I just followed the protonmail records steps with the now CLI and worked perfectly.
